backend/src/Core/CommandLine.cs

```diff
@@ -161,30 +162,29 @@ private static string ValidateDependencies(CommandResult result)
                     }
                 }
 
-                // TODO: Enable once CosmosDB is setup
                 // validate secrets volume
-                // if (appType == AppType.App)
-                // {
-                //     if (string.IsNullOrWhiteSpace(secrets))
-                //     {
-                //         msg += "--secrets-volume cannot be empty\n";
-                //     }
-                //     else
-                //     {
-                //         try
-                //         {
-                //             // validate secrets-volume exists
-                //             if (!Directory.Exists(secrets))
-                //             {
-                //                 msg += $"--secrets-volume ({secrets}) does not exist\n";
-                //             }
-                //         }
-                //         catch (Exception ex)
-                //         {
-                //             msg += $"--secrets-volume exception: {ex.Message}\n";
-                //         }
-                //     }
-                // }
+                if (appType == AppType.App)

```

Comment:
Let's remove any flags that we know we aren't going to use from the command line processing.  The AppType value was used in NGSA to run it as a webapi or dataservice so we could show a split.  We need to remove that from this code base.  So, we must remove AppType and any logic and such around it.  For example, this conditional will no longer be needed.